src/main/java/edu/byu/cs/autograder/Grader.java

```diff
@@ -360,7 +360,8 @@ private Submission saveResults(Rubric rubric, int numCommits, int numDaysLate, f
                 score,
                 numCommits,
                 notes,
-                rubric
+                rubric,
+                DaoService.getUserDao().getUser(netId).role() == User.Role.ADMIN
```

Comment:
Rather than fetching the user from the db twice, I think it'd be better to do it once before the if statement

Reply:
I'm a fan of making fewer calls to the Database. I'm not sure what the second call you're referring to. In the saveResults() function, there are two calls to the Database, but they're for different tables. One is `DaoService.getQueueDao().get(netId)` and the other is `DaoService.getUserDao().getUser(netId)`. I don't think we call the database for the user any other place in this process, but I could be wrong

Reply:
Yup, you're right, my comment doesn't make much sense. When I looked at this on GitHub I didn't notice that these changes were happening in different functions
![image](https://github.com/softwareconstruction240/autograder/assets/10139415/644c3702-5f20-46fa-964a-c7bd226ee52f)

Reply:
Nevermind 😅

Reply:
❤️